config.py

```diff
@@ -1988,6 +2013,23 @@ def gen_log_kwargs(
     raise ValueError(msg)
 
 
+def _validate_contrasts(contrasts):
+    for contrast in contrasts:
+        if isinstance(contrast, tuple):
+            if len(contrast) != 2:
+                raise ValueError("Contrasts' tuples MUST be two conditions")
+        else:
+            if "name" not in contrast.keys():
+                raise ValueError("One contrast is missing a name")
+            if "conditions" not in contrast.keys():
+                raise ValueError("One contrast is missing a condition list")
+            if "weights" not in contrast.keys():
+                raise ValueError("One contrast is missing a weight list")
+            if len(contrast["conditions"]) != len(contrast["weights"]):
+                raise ValueError(f"Contrast {contrast['name']} has an"
+                                 f"inconsistent number of conditions/weights")
```

Comment:
Can we avoid these checks by just trying to cast the passed `dict` to an `ArbitraryContrast`?

Reply:
Can't find a straightforward way to bend python's type system to my will and do that.... Still streamlined that check